File: app/src/main/java/com/michaeljordanr/memesounds/Audio.java
package com.michaeljordanr.memesounds;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Audio implements Serializable {
    @Expose
    private int id;

    @Expose
    @SerializedName("audio_name")
    private String audioName;

    @Expose
    @SerializedName("audio_description")
    private String audioDescription;

    @Expose
    @SerializedName("audio_thumb")
    private String audioThumb;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getAudioName() {
        return audioName;
    }

    public void setAudioName(String audioName) {
        this.audioName = audioName;
    }

    public String getAudioDescription() {
        return audioDescription;
    }

    public void setAudioDescription(String audioDescription) {
        this.audioDescription = audioDescription;
    }

    public String getAudioThumb() {
        return audioThumb;
    }

    public void setAudioThumb(String audioThumb) {
        this.audioThumb = audioThumb;
    }
}

File: app/src/main/java/com/michaeljordanr/memesounds/Audio.kt
package com.michaeljordanr.memesounds

import com.squareup.moshi.Json

data class Audio(
        @field:Json(name = "audio_id")
        var id: Int,
        @field:Json(name = "audio_name")
        var audioName: String,
        @field:Json(name = "audio_description")
        var audioDescription: String,
        @field:Json(name = "audio_thumb")
        var audioThumb: String
)
